starknet_py/utils/data_transformer/errors.py

```diff
@@ -0,0 +1,6 @@
+class CairoSerializerException(TypeError):
+    """Exception thrown by CairoSerializer"""
+
+
+class InvalidDataException(ValueError):
```

Comment:
So this is an error for `to_python` methods? Maybe we could name it `DeserializationError` and the other one `SerializationError`?

Reply:
I see that it is not the case. Users won't understand their meaning.